modular_skyrat/master_files/code/datums/traits/negative.dm

```diff
@@ -79,5 +79,5 @@
 	alert_type = null
 
 /datum/status_effect/sensitivesnout/get_examine_text()
-	return span_warning("[owner.p_their(TRUE)] snout is rather bappable...")
+	return span_warning("[owner.p_Their())] snout is rather bappable...")
```

Comment:
```suggestion
	return span_warning("[owner.p_Their()] snout is rather bappable...")
```

Reply:
Also since this comes from srat and is an error there you would be better off pushing this upstream

Reply:
This is an us thing, rodent serv doesn't have the trait do anything at all because of various reasons.

Reply:
Either way they still have it in its broken form